### PR TITLE
Styled accordion

### PR DESCRIPTION
**Pull request by kyle-beattie**
Created basic styles for accordions and fixed blog page layout so that opening an accordion details doesn't break the layout.

[See the story](https://vimaly.com/#j8mqm/t/www_seo_1.5._Get_a_styled_accordion_block_working_.../9JfojOLll8WSrG48n)

### CODE REVIEW

File: source/css/blog/_page.blog_page.scss [hunk 1]
```diff
@@ -6,14 +6,6 @@
   .blog_content_container {
     @media (min-width: 1440px) {
       max-width: 950px;
-      position: relative;
-      left: -30px;
     }
   }
 }
-
-.blog_content_contained {
```
Comment (kylorhall) [inline source/css/blog/_page.blog_page.scss:15]: 🙈  Probably needs a bit of QA, unfortunately…hopefully it's perfect!

File: source/css/elements/_accordion.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
+details {
+  overflow: hidden;
+  display: block;
+  width: 100%;
+}
+
+summary {
+  padding: 20px;
+  margin-bottom: 1px;
```
Comment (kylorhall) [inline source/css/elements/_accordion.scss:9]:
```suggestion
```

**Comment by kyle-beattie** [comment]
This is on staging now: https://staging-www.sharesight.com/blog/markdown-collapsible-section-test/

File: source/css/elements/_accordion.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  background-color: $background_offset_white;
+  cursor: pointer;
+  font-weight: 600;
+  color: $colour_base;
+  @include fontSize(18px, 2.125);
+  line-height: 2.125;
+}
```
Comment (kylorhall) [inline source/css/elements/_accordion.scss:16]: What is `$gutter`…?  I don't even know.  I _think_ `padding: 20px` is a bit much—I'd aim for `padding: 15px` personally here (suggested as `padding: $gutter;`.

1. Does `@include fontSize(18px, 2.125);` not include `line-height`?
2. The margin between is lacking.  Maybe my suggestion is too much/too little?  They should definitely feel "grouped" in my book.
Screen Shot 2021-05-25 at 12 34 25 PM

Also just keen if you get this same feedback from Sam Straun 😉 

```suggestion
summary {
  padding: $gutter;
  background-color: $background_offset_white;
  cursor: pointer;
  font-weight: 600;
  color: $colour_base;
  @include fontSize(18px, 2.125);
  line-height: 2.125;
  
  & + & {
    margin-top: $gutter/2;
  }
}
```

**Review by kylorhall — CHANGES_REQUESTED** [review]
Almost 💯, just design feedback actually..

File: source/css/elements/_accordion.scss (hunk 3 above)
Comment (kyle-beattie) [reply source/css/elements/_accordion.scss:16]: Ha, `$gutter` is `20px`

**Review by kylorhall — APPROVED** [review]
I'll call it approved.

Ping Angela/David for "please try this somewhere…" and then Sophie for generic QA as you please..

**Comment by kyle-beattie** [comment]
@sophie-joe has already done some QA on [the story](https://vimaly.com/#j8mqm/t/www_seo_1.5._Get_a_styled_accordion_block_working_.../9JfojOLll8WSrG48n) and she's pinged David 👍 . I asked Sam for some feedback but haven't heard back yet.

**Pull request merged**